tutorials/appengine-serve-machine-learning-model/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Serve Machine Learning Model on App Engine Flex
+description: Learn how to serve a trained machine learning model with Google App Engine flex environment.
+author: dizcology
+tags: App Engine, Cloud Endpoints, Machine Learning
+date_published: 
+---
+This tutorial takes a deeper look at the sample app [Model serve][modelserve].  It helps you build your own service serving a trained machine learning model for online prediction.
+
+## Objectives
+
+1. Deploy a service with [Google Cloud Endpoints][endpoints].
+1. Deploy a Python app on [Google App Engine][appengine] which loads a trained machine learning model.
+1. Send requests to the service and get responses.
+
+## Before you begin
+
+Follow the links in the [Requirements section][requirements] to install Google Cloud Platform SDK and enable the APIs for App Engine, Cloud Endpoints, and Google Cloud Storage.
+
+## Overview
+
+So you trained a machine learning model.  Now what?
+
+If the model’s performance is good enough, consider deploying it as a service to a production system where one or more clients can use it.  Some possible scenarios include:
+
+- The model needs to process requests in real time when the users are interacting with your web application while at the same time batch process interaction logs you have stored previously in a database.
+
+- The model's output is used by multiple other machine learning models in your application.
+
+[App Engine][appengine] offers rolling update, networking, and auto scaling.
+
+[Cloud Endpoints][endpoints] helps you monitor and manage permission and quota of the service's consumers.
```

Comment:
[Cloud Endpoints][endpoints] helps you monitor the service's consumers, as well as manage their permissions and quotas.